src/games/strategy/performance/PerfTimer.java

```diff
@@ -1,8 +1,12 @@
 package games.strategy.performance;
 
-import java.io.Closeable;
+import java.util.prefs.Preferences;
 
-public class PerfTimer implements Closeable {
```

Comment:
No, we want this to be Closeable. Whenever we use a timer, it should be closed in a 'finally' block to make sure we always close the timer. There was a reason for there being two classes, Perf provided access to a PerfTimer, which was then auto-closeable.

If you could not tell,I do not agree with the combination of the two classes. The role of one class was a factory, the other one is a timer.